sound/soc/codecs/rt715-sdca.h

```diff
@@ -115,7 +115,6 @@ struct rt715_sdw_stream_data {
 enum {
 	RT715_AIF1,
 	RT715_AIF2,
-	RT715_AIFS,
```

Comment:
can we add the same cleanup for rt715.h? (as a separate patch)

Reply:
@plbossart  Please refer new PR #2523 , thanks.